cosmic_ray/operators/comparison_operator_replacement.py

```diff
@@ -48,7 +48,7 @@ def _all_ops(from_op):
 
 def _rhs_is_none_ops(from_op):
     for key, value in _RHS_IS_NONE_OPS.items():
-        if isinstance(from_op, key):
+        if from_op is key:
```

Comment:
Is there a case for using issubclass() in this and other locations? That would seem to be more equivalent to the original isinstance() call, or do we need an exact type check?

Reply:
From what I can see, there are no inheritance relationships we'd want to capture here. I went with `is` since it's more precise and less likely to result in surprises.